README.md

```diff
@@ -1102,6 +1109,33 @@ Unimplemented `docker-compose ps` (V1) flags: `--quiet`, `--services`, `--filter
 
 Unimplemented `docker compose ps` (V2) flags: `--format`, `--status`
 
+## IPFS management
+
+### :nerd_face: nerdctl ipfs registry up
+Start read-only local registry backed by IPFS.
+See [`./docs/ipfs.md`](./docs/ipfs.md) for details.
+
+Usage: `nerdctl ipfs registry up [OPTIONS]`
+
+Flags:
+- :nerd_face: `--listen-registry-port`: Listening port on localhost (default `5050`)
+
+### :nerd_face: nerdctl ipfs registry down
+Stop and remove read-only local registry backed by IPFS.
+See [`./docs/ipfs.md`](./docs/ipfs.md) for details.
+
+Usage: `nerdctl ipfs registry down`
+
+### :nerd_face: nerdctl ipfs registry serve
+Serve read-only registry backed by IPFS on localhost.
+Use `nerdctl ipfs registry up`.
+
+Usage: `nerdctl ipfs registry serve [OPTIONS]`
+
+Flags:
+- :nerd_face: `--ipfs-address`: Multiaddr of IPFS API (default is pulled from `$IPFS_PATH/api` file. If `$IPFS_PATH` env var is not present, it defaults to `~/.ipfs`).
+- :nerd_face: `--listen-registry-port`: Listening port on localhost (default `5050`).
```

Comment:
Can we make this like `--listen-registry=localhost:5050` so that we can specify non-localhost address too?

Reply:
Fixed.